src/core/control/Control.cpp

```diff
@@ -78,7 +78,7 @@ Control::Control(GApplication* gtkApp, GladeSearchpath* gladeSearchPath): gtkApp
 
     this->applyPreferredLanguage();
 
-    TextView::setDpi(settings->getDisplayDpi());
+    xoj::View::TextView::setDpi(settings->getDisplayDpi());
```

Comment:
I don't like how we have what is effectively a global variable controlling the dpi. How involved would it be to remove the global variable?

Reply:
You mean removing the variable xoj::view::TextView::DPI, right?
This is a little bit more complicated than expected. We'd need TextView to have a pointer to the Settings instance, and to do so, we'd need every class creating a TextView to have a pointer to the Settings.
The worst case is model/Text (needs TextView for calcSize(), see discussoin above).

I think we should do a deeper refactoring of this class, but I'd leave is like that for now.

Reply:
We should consider a singleton approach for the Settings class. That way we would not need to forward it through half the application.